### PR TITLE
feat: add a cross-validation step

### PR DESCRIPTION
**Pull request by robincourant**
- create new file with a method used for the cross-validation.
- add a method to convert a dataframe into a design matrix and a label 
vector (numpy arrays).
- Fix wrong return statements in data-preparation.
- Add a new argument in the CLI and modify the doc according to that 
change.

### CODE REVIEW

File: data_evaluation.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+from typing import Tuple
+
+import numpy as np
+from sklearn.base import BaseEstimator, clone
+from sklearn.model_selection import KFold
```
Comment (yakoubix) [inline data_evaluation.py:5]: Sort imports in alphabetical order

File: data_preparation.py [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import os
+from typing import Tuple
 
+import numpy as np
 import pandas as pd
 from pandas.api.types import is_integer_dtype, is_float_dtype
```
Comment (yakoubix) [inline data_preparation.py:6]: Sort imports in alphabetical order

**Review by yakoubix — CHANGES_REQUESTED** [review]
Clean code with clear comments and documentation. The only raised point was the alphabetical sort of the imports, which becomes important when the code gets bigger.
